godotproject/PlayerTongue.gd

```diff
@@ -10,6 +10,7 @@ export (float) var max_shoot_dist = 400 * 0.6
 const COLLISION_LAYER_TONGUEABLE         = 0x3
 const COLLISION_LAYER_TONGUE_CAN_DAMAGE  = 0x6
 const COLLISION_LAYER_TONGUE_CAN_COLLECT = 0x7
+const COLLISION_LAYER_TONGUE_CAN_SLIDE   = 0x9
```

Comment:
I don't think my new stuff uses this. Can we remove?

Reply:
oh right, good catch thank you